test/e2e/test_cli.py

```diff
@@ -234,52 +231,6 @@ def test_excel_to_json_list(self) -> None:
         self.assertListEqual(output_actual, output_expected)
         out_file.unlink()
 
-    def test_excel_to_json_resources(self) -> None:
-        excel_file = Path("testdata/excel2json/excel2json_files/test-name (test_label)/resources.xlsx")
-        out_file = self.testdata_tmp / "_out_resources.json"
-        self._make_cli_call(f"dsp-tools excel2resources '{excel_file.absolute()}' {out_file.absolute()}")
-        with open(out_file, encoding="utf-8") as f:
-            output_actual = json.load(f)
-        with open("testdata/excel2json/resources-output-expected.json", encoding="utf-8") as f:
-            output_expected = json.load(f)
-        self.assertListEqual(output_actual, output_expected)
-        out_file.unlink()
-
-    def test_excel_to_json_properties(self) -> None:
-        excel_file = Path("testdata/excel2json/excel2json_files/test-name (test_label)/properties.xlsx")
-        out_file = self.testdata_tmp / "_out_properties.json"
-        self._make_cli_call(f"dsp-tools excel2properties '{excel_file.absolute()}' {out_file.absolute()}")
-        with open(out_file, encoding="utf-8") as f:
-            output_actual = json.load(f)
-        with open("testdata/excel2json/properties-output-expected.json", encoding="utf-8") as f:
-            output_expected = json.load(f)
-        self.assertListEqual(output_actual, output_expected)
-        out_file.unlink()
```

Comment:
These 2 are already tested in `test/unittests/test_excel2json/test_resources.py` and `test/unittests/test_excel2json/test_properties.py`. But those 2 are difficult to maintain. So I'm not 100% sure if it is worth getting rid of these 2 here (and of the testdata files), if in turn we have to maintain 2 difficult unit tests.